apps/ledger-live-mobile/src/components/Web3AppWebview/helpers.ts

```diff
@@ -394,6 +411,7 @@ function useUiHook(): Partial<UiHook> {
           },
         });
       },
+      "transaction.broadcast": () => {},
```

Comment:
Yes I'll add a comment

This is done to avoid the previous usage of `Partial` on this type and avoid optional chaining to call the methods in the `useDappLogic` hook